### DISCUSSION TITLE
Looking for suggestions on how to order board, case and parts

### DISCUSSION THREAD

**Discussion by mircsicz** [Zeppelin500/MBusino#17] [General]
I was about to order on PCBway.

Already got two cases (two are 30ct more than just one!), but I can't figure how to get a quote for a PCB from the Fritzing file...

And also if possible I'ld like to order the other part's but didn't quite get the quote about Omnibus:
> To save mony, I place omnibus orders for all parts.

for the MBus Board I'm about to order and D1 Mini's I got on the 'shelf'...

**Comment by Zeppelin500** [comment]
Write a mail to mbusinolib@gmail.com. I can help.
